Compiling shim on ClearLinux gives error

Compiling shim on clearlinux version 17650, gives the following error```
```
src/utils.c:67:2: error: ‘SIGUNUSED’ undeclared here (not in a function); did you mean ‘SI_USER’?
  SIGUNUSED,
  ^~~~~~~~~
```
Turns out that this constant has been removed in glib2.6 that clearlinux 17650 uses.
Ref: https://sourceware.org/git/?p=glibc.git;a=blob;f=NEWS;h=dd381f89308d850e0513cae935fb02adb094845a;hb=HEAD#l198

--- a/src/utils.c
+++ b/src/utils.c
@@ -64,7 +64,9 @@ int shim_signal_table[] = {
 	SIGIO,               /* I/O now possible */
 	SIGPWR,              /* Power failure restart */
 	//SIGSYS,            /* Bad system call */
+#ifdef SIGUNUSED
 	SIGUNUSED,
+#endif
 	0,
 };
 

